ArduCopter/defines.h

```diff
@@ -58,7 +58,8 @@ enum tuning_func {
     TUNING_RATE_MOT_YAW_HEADROOM =      55, // motors yaw headroom minimum
     TUNING_RATE_YAW_FILT =              56, // yaw rate input filter
     UNUSED =                            57, // was winch control
-    TUNING_SYSTEM_ID_MAGNITUDE =        58  // magnitude of the system ID signal
+    TUNING_SYSTEM_ID_MAGNITUDE =        58, // magnitude of the system ID signal
+    TUNING_POS_CONTROL_ANGLE_MAX =      59  // position controller maximum angle
```

Comment:
This new enum value should also be added to the TUNE parameter definition I think.